### ISSUE TITLE
Studiomdl error?

### ISSUE THREAD

**Issue by valina354** [TheOverfloater/pathos-public#47]
im trying to compile model as part of me trying to figure out any work around to fact models dont have lightmaps, and im getting this error
![image](https://github.com/user-attachments/assets/a85d784a-8bcf-408b-b137-25a6f7205a4a)

am i doing something wrong? heres my  qc here file
![image](https://github.com/user-attachments/assets/2dac5cee-d296-4af1-af7d-1b1e03a29948)

and folder stuff is in
![image](https://github.com/user-attachments/assets/9cd31e19-85be-4eb0-a224-219a6ec48ed0)

**Comment by valina354** [comment]
it seems to be something related to texture? i changed name of the smd and qc to check and still that error
![image](https://github.com/user-attachments/assets/bf54d519-45e4-44fe-80eb-db9aa8f29660)
and removing cdtexture just for testing gives this
![image](https://github.com/user-attachments/assets/9642a3f8-7d9d-44d7-8128-8fdb774e82eb)

**Comment by valina354** [comment]
wait i used the -t to specify .bmp and it seems to have worked but gives texture too large, i will lower it and check if it fixed the errors
![image](https://github.com/user-attachments/assets/b9df7b4d-8cb2-486b-b411-28f3beef69a0)
this is one of things i want to fix when the studiomdl rewrite is done so i can modify compiler to raise this because it looks bad in editor i think due to needing way lower res then it actually is

**Comment by valina354** [comment]
ok so lowering res fixed the compiling, but after it compiled the vbm has no data for some reason, its only 1kb of just this
![image](https://github.com/user-attachments/assets/d9f18131-264e-4e84-9e04-22a8c684af00)

do you know what might be causing this? @TheOverfloater 

im not sure if its related to reference_frames.smd, im not sure what it means, i just did this
![image](https://github.com/user-attachments/assets/9c7c88e7-036e-4745-ac00-7b4ef330e77c)

**Comment by valina354** [comment]
heres output from the compiler, its detecting 0 vertexes and indexes i think thats issue, but idk why, 
![image](https://github.com/user-attachments/assets/906e75a4-6152-43b1-9f71-85018d77bda2)

**Comment by valina354** [comment]
i tried not setting -l but it didnt do anything

**Comment by TheOverfloater** [comment]
Please send the folder with the .qc and all source files to doommusic666@hotmail.com, and I will take a look.

**Comment by valina354** [comment]
ok, so i kinda changed the model to another one but still same thing so i will send all the files i guess

**Comment by valina354** [comment]
sent, its using my company's email btw so you dont get confused

**Comment by valina354** [comment]
have you checked this out btw or not since im gonna try to implement the model lightmapping now and be able to have a model to test yet i cant get it compiled @TheOverfloater

**Comment by TheOverfloater** [comment]
Numerous issues I found and one safeguard applied in [b0a61c4](https://github.com/TheOverfloater/pathos-public/commit/b0a61c432d724ae4523d793efdf24c2db0eac686):
 - Your reference_frame file is missing the first frame of the skeleton. You only need to delete everything after "triangles" and keep the rest preceeding it when creating a reference_frame.smd file.
 - Your export tools still put a 0 for the number of weights at the end of the triangle vertex data, which caused issues. I added a safeguard for this.
 - Your model does not have any animations defined with $sequence, which is an issue.
